### PR TITLE
[ceph] Add more microceph commands

### PR DESCRIPTION
**Pull request by pponnuvel**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?
- [ ] Are all passwords or private data gathered by this PR [obfuscated](https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin#how-to-prevent-collecting-passwords)?

### CODE REVIEW

File: sos/report/plugins/ceph_common.py [hunk 1]
```diff
@@ -95,12 +95,22 @@ def setup(self):
...
+            cmds = [
+                'client config list',
+                'cluster list',
+                'cluster config list',
+                # exclude keyrings from the config db
+                'cluster sql \'select * from config where key NOT LIKE \
+                    \"%keyring%\"\'',
+                'disk list',
+                'log get-level',
+                'status'
```
Comment (arif-ali) [inline sos/report/plugins/ceph_common.py:109]: lgtm

minor nit pick, could add a comma at the  end :). When we add new commands, we just add new commands on new line

**Review by arif-ali — APPROVED** [review]

File: sos/report/plugins/ceph_common.py (hunk 1 above)
Comment (pponnuvel) [reply sos/report/plugins/ceph_common.py]: ack, done

**Review by arif-ali — APPROVED** [review]

File: sos/report/plugins/ceph_common.py [hunk 2]
```diff
@@ -95,12 +95,22 @@ def setup(self):
...
-            self.add_cmd_output("microceph status", subdir="microceph")
             self.add_cmd_output("snap info microceph", subdir="microceph")
-            # Collect all but exclude any keyrings
-            self.add_cmd_output("microceph cluster sql 'select * from config \
-                                 where key NOT LIKE \"%keyring%\"'",
-                                subdir="microceph")
+
+            cmds = [
+                'client config list',
+                'cluster config list',
```
Comment (dnegreira) [inline sos/report/plugins/ceph_common.py:102]: @pponnuvel is there any private information on config list that should be censored?

Comment (pponnuvel) [reply sos/report/plugins/ceph_common.py:102]: @dnegreira  This gets the configuration settings for daemons. This is sort of the equivalent of "ceph config show" in the standard Ceph (which we already collect for Ceph). This currently supports just two: `cluster_network`, and `osd_pool_default_crush_rule`. And don't expect any confidential here in the future either.

**Review by TurboTurtle — APPROVED** [review]

**Pull request merged**